setup.py

```diff
@@ -204,8 +205,9 @@ def run(self):
 
 
 extras = {}
-extras["quality"] = deps_list("urllib3", "isort", "ruff", "hf-doc-builder")
-extras["docs"] = deps_list("hf-doc-builder")
+# `hf-doc-builder` has a dependency on `black`. See huggingface/doc-builder#434.
+extras["quality"] = deps_list("urllib3", "isort", "ruff", "hf-doc-builder", "black")
+extras["docs"] = deps_list("hf-doc-builder", "black")
```

Comment:
Do we still need to add `black` as a hidden dependency that they added as a required dependency per this PR (https://github.com/huggingface/doc-builder/pull/434) and had a new release three weeks ago?

Reply:
Yeah we don't probably. Will remove.